recipes/cyipopt/meta.yaml

```diff
@@ -21,26 +21,20 @@ build:
 
 requirements:
   build:
-    - toolchain
     - gcc
     - pkg-config  # [osx]
     - python
     - ipopt
     - setuptools
     - cython
-    - numpy  # [not py36]
-    # NOTE : During the build stage NumPy 1.9.3 is downloaded instead of the
-    # latest version. Since there is no NumPy 1.9.3 package for Python 3.6 in
-    # defaults or conda-forge, this causes the build to fail. The following
-    # forces a newer version to be used.
-    - numpy >=1.11  # [py36]
+    - numpy 1.11
```

Comment:
I will take a look later today but in theory the "right" approach is to use `numpy x.x`.